src/app/(marketing)/profile/_components/ActivityChart.tsx

```diff
@@ -0,0 +1,100 @@
+"use client"
+
+import {
+  ChartConfig,
+  ChartContainer,
+  ChartTooltip,
+  ChartTooltipContent,
+} from "@/components/ui/Chart"
+import { Bar, BarChart, XAxis } from "recharts"
```

Comment:
Same here, we should use visx, but we should just remove the component now, we'll handle it when it's actually used